Automattic/jetpack
# Forms: Increase form fields padding based on user-defined border-radius

Fixes #28776

## Proposed changes:
* Increase form fields padding based on user-defined border-radius. More context on #28776

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a Post and add a Form block
* Increase the border-radius from a field using the Sidebar settings
* Check if the text/placeholder isn't hidden even when using a big border-radius.
